js/timers.js

```diff
@@ -0,0 +1,70 @@
+// A collection of globally accessible timers
+
+const timers = {
+  poison: function() {
+    //poison elements
+    const poisonRightHalf = document.querySelector('#poison-filler');
+    const poisonLeftHalf = document.querySelector('#poison-spinner');
+    const poisonMask = document.querySelector('#poison-mask');
+
+    //health elements
```

Comment:
I don't like the idea of having the health elements within the poison timer. This should be separated by calling a method on player that takes in a resulting damage done by poison as a parameter. the method of player should be the one handling the logic of taking the damage and be in charge of rerendering the hp bar.